static/js/routes/common.min.js

```diff
@@ -7063,20 +7063,20 @@ return je(this,function(t,n,r){var i;return oe.isWindow(t)?t.document.documentEl
 
 }).call(this,typeof global !== "undefined" ? global : typeof self !== "undefined" ? self : typeof window !== "undefined" ? window : {})
-},{"/Users/bradforda/Projects/cfgov-refresh/node_modules/jquery/dist/jquery.js":34}],72:[function(require,module,exports){
+},{"/Users/wilsong/repos/cfpb/cfgov-refresh/node_modules/jquery/dist/jquery.js":34}],72:[function(require,module,exports){
```

Comment:
@Scotchester it's browserify-shim's great idea https://github.com/cfpb/cfgov-refresh/issues/560